workflow/scripts/neighborhoods.py

```diff
@@ -426,7 +426,17 @@ def run_count_reads(target, output, bed_file, genome_sizes, use_fast_count):
         raise ValueError(
             "File {} name was not in .bam, .tagAlign.gz, .bw".format(target)
         )
-
+    double_sex_chrom_counts(output)
+    
+
+def double_sex_chrom_counts(output):
+    # Double the count values for sex chromosomes to make it seem 
+    # like they have 2 copies
+    awk_command = r'''awk 'BEGIN {FS=OFS="\t"} (substr($1, length($1)) == "X" || substr($1, length($1)) == "Y") { $4 *= 2 } 1' '''
```

Comment:
Not sure if we want to support cases where chromosome names don't have 'chr' prefixed, so I opted to just check if the last character is X or Y